charm/layer-candid/reactive/candid.py

```diff
@@ -89,7 +89,10 @@ def write_config_file():
         config["public-key"] = lc["public-key"]
     if cc["redirect-login-whitelist"]:
         config["redirect-login-whitelist"] = \
-            cc["redirect-login-whitelist"].split(",")
+            _parse_list(cc["redirect-login-whitelist"])
+    if cc["redirect-login-trusted-domains"]:
```

Comment:
Is this key guaranteed to be defined, even in a charm upgrade scenario?

Reply:
I'm pretty sure it is.